src/main/java/net/minecraftforge/fml/common/gameevent/PlayerEvent.java

```diff
@@ -34,34 +34,6 @@ private PlayerEvent(EntityPlayer player)
         this.player = player;
     }
 
-    public static class ItemPickupEvent extends PlayerEvent {
```

Comment:
note: this FML item pickup event isn't marked deprecated, but it's been duplicated by the forge item pickup event for a while. Does anyone still need something from this FML event that the Forge one doesn't let you do?

Reply:
The comments in #3419 suggest that the two events aren't exactly equivalent.

(e.g. https://github.com/MinecraftForge/MinecraftForge/issues/3419#issuecomment-307020636, https://github.com/MinecraftForge/MinecraftForge/issues/3419#issuecomment-324213982)

Reply:
Should probably just leave it for a later commit to clean up both events.